src/builtin_cd.c

```diff
@@ -0,0 +1,58 @@
+#include "bash_ops.h"
+#include "env_access.h"
+#include "filesystem_traversal.h"
+#include "ft_printf.h"
+#include "path.h"
+#include <stdlib.h>
+
+#define DIR_HAS_CHANGED 0
+
+static int 	__go_to_prev_dir()
+{
+	return (fs_change_dir(env_get_prev_dir()));
+}
+
+static int	__go_to_home()
+{
+	return (fs_change_dir(env_get_home()));
+}
+
+static t_bool	__update_dir_change(t_command *cmd)
+{
+	return (env_set_s("OLDPWD", env_get_s("PWD")) && 
+			env_set_s("PWD", cmd->argv->argv[1]));
```

Comment:
not much use of getters in env_access if you set them manually 

also is there a reason why you are passing t_cmd and not char * ?